### DISCUSSION TITLE
Notification from Google Play: Your app(s) are vulnerable to Intent Redirection.

### DISCUSSION THREAD

**Discussion by Klug76** [airsdk/Adobe-Runtime-Support#1487] [Q&A]
Hi!
After submitting my AIR-based game, I received this message:
=================
We reviewed %app name%, and found that your app uses software that contains security vulnerabilities for users. Apps with these vulnerabilities can expose user information or damage a user’s device, and may be considered to be in violation of our Malicious Behavior policy.

Below is the list of issues and the corresponding APK versions that were detected in your recent submission. Please migrate your apps to use the updated software as soon as possible and increment the version number of the upgraded APK.

Vulnerability
Intent Redirection
Your app(s) are vulnerable to Intent Redirection.

Deadline to fix
March 22, 2022
=================
It is clear that the problem is not in the as3 code, but either in the native extensions, or in the AIR itself.
I would be glad to any suggestions on how to find the source of the problem.

**Comment by ajwfrost** [comment]
Hi

Does it mention the vulnerable class(es)? The below link shows some screenshots where those are available although there's a comment also saying that this info is now hidden unless you email Google support:
https://stackoverflow.com/questions/56175958/your-app-contains-an-intent-redirection-vulnerability

I've not heard of this at all before so would assume it's in an ANE rather than the AIR runtime itself, but it's definitely one to check...

thanks

**Comment by Klug76** [comment]
Yes, but now there is no alert tab in the new developer console, and worse, I can't find where the email support is. :(

**Reply by Klug76** [reply]
After reviewing all the tabs, I found what's the matter!
```
com.facebook.CustomTabMainActivity->onCreate
sv:deadline:03/22/2022
```
so this is freshplanet facebook ane (slightly outdated)

**Reply by letieu** [reply]
Did you know hơe to fix it ?

**Reply by Klug76** [reply]
Fix here:
<activity android:name="com.facebook.CustomTabMainActivity" **android:exported="false"** />

**Reply by letieu** [reply]
I have try this line before but it not work, and I try remove all facebook sdk, but I still get same notify from google play :(

**Reply by marchbold** [reply]
@letieu what version of air are you using? It's potentially caused by an older air SDK.

**Reply by letieu** [reply]
react-native: 0.60
I was remove react-native-fbsdk from my project and all com.facebook android sdk from android folder

**Reply by archon810** [reply]
Do you remember what tab you found it under in the end, in the new UI?
